From 3c72e4cd6fa29a99f939d439454321b839f036ea Mon Sep 17 00:00:00 2001
From: Li Zeng <li.zeng@intel.com>
Date: Sat, 27 Oct 2012 10:00:16 +0800
Subject: [PATCH 25/35] [PORT FROM JBMR0] Add a flag
 OMX_INTEL_COLOR_FormatYUV420PackedSemiPlanar_Tiled

BZ: 70693

Add a flag OMX_INTEL_COLOR_FormatYUV420PackedSemiPlanar_Tiled

Change-Id: I38da279d822346dddcd6805df2686992a654ed36
Orig-Change-Id: I59194920852b745c76d22d6c22f01b8f4cc4713b
Signed-off-by: Li Zeng <li.zeng@intel.com>
Reviewed-on: http://android.intel.com:8080/83203
Tested-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/media/openmax/OMX_IVCommon.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/media/openmax/OMX_IVCommon.h b/include/media/openmax/OMX_IVCommon.h
index de40dee..08fb11b 100755
--- a/include/media/openmax/OMX_IVCommon.h
+++ b/include/media/openmax/OMX_IVCommon.h
@@ -159,6 +159,7 @@ typedef enum OMX_COLOR_FORMATTYPE {
     OMX_COLOR_FormatAndroidOpaque = 0x7F000789,
     OMX_TI_COLOR_FormatYUV420PackedSemiPlanar = 0x7F000100,
     OMX_INTEL_COLOR_FormatYUV420PackedSemiPlanar =  0x7FA00E00,
+    OMX_INTEL_COLOR_FormatYUV420PackedSemiPlanar_Tiled =  0x7FA00F00,
     OMX_QCOM_COLOR_FormatYVU420SemiPlanar = 0x7FA30C00,
     OMX_QCOM_COLOR_FormatYUV420PackedSemiPlanar64x32Tile2m8ka = 0x7FA30C03,
     OMX_COLOR_FormatMax = 0x7FFFFFFF
-- 
1.7.9

